_docs/torneos/ab21.md

```diff
@@ -18,21 +18,97 @@ tags:
 
 ## Charla informativa
 
-**Viernes 27 de septiembre, 05:30 pm**. 
+**Viernes 27 de agosto, 05:30 pm**. 
 
 Únete mediante el siguiente enlace a [Google Meet](https://meet.google.com/kgh-dbhz-bbu).
 
 ## Equipos
 
-*por determinarse*
 
-## Clasificatorias
+|                    Equipo                    |    Miembros del equipo |
+|:--------------------------------------------:|:----------------------:|
+| 1p0l1t0             |        ***********       |
+| Alonso              |        ***********       |
+| BlackCat            |        ***********       |
+| cualquiera          |        ***********       |
+| delorean            |        ***********       |
+| Dretcm              |        ***********       |
+| dubinator           |        ***********       |
+| fedeo_Molitalia     |        ***********       |
+| flechita_veloz      |        ***********       |
+| GalagaRebirth       |        ***********       |
+| iguana_dye          |        ***********       |
+| LaUvaLoca           |        ***********       |
+| leroyG5             |        ***********       |
+| Los_Reales_Gs       |        ***********       |
+| Mark-Lex21          |        ***********       |
+| sospechoso          |        ***********       |
+| TheBotLess          |        ***********       |
+| thisisabot          |        ***********       |
+| TILIN               |        ***********       |
 
-*por determinarse*
+## Clasificatorias resultados 16/09/21
+
+|Equipo            | Ronda 1 | Ronda 2 | Ronda 3 | Ronda 4 | Ronda 5 | Ronda 6 | Ronda 7 | Ronda 8 | Ronda 9 | Ronda 10 | Ronda 11 | Ronda 12 | Ronda 13 | Ronda 14 | Puntaje |
```

Comment:
Please, make `Puntaje` the second column.

Reply:
do it